src/internal_events/reduce.rs

```diff
@@ -9,3 +10,26 @@ impl InternalEvent for ReduceStaleEventFlushed {
         counter!("stale_events_flushed_total").increment(1);
     }
 }
+
+#[derive(Debug)]
+pub struct ReduceAddEventError {
+    pub error: PathParseError,
+}
+
+impl InternalEvent for ReduceAddEventError {
+    fn emit(self) {
+        error!(
+            message = "Event could not be reduced.",
```

Comment:
```suggestion
            message = "Event field could not be reduced.",
```

The reduce is field by field so I think it is more accurate to say the specific field couldn't be reduced since earlier fields may have already been reduced.